src/styles/components/calendar/calendar-style.scss

```
@@ -0,0 +1,51 @@
+@import "calendar-variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  gap: $days-gap;
+  width: $calendar-width;
+  padding: $calendar-padding;
+
+  &__day {
+    box-sizing: border-box;
+    border: $days-border;
+    background-color: $days-background;
+    width: $day-size;
+    height: $day-size;
+    text-align: center;
+    font-size: $days-font-size;
+    line-height: $day-size;
+    transition-duration: 0.5s;
```

By default transition is use for all and it's may be prettu hard for browser. You shold set for what properties transition will be applied
```suggestion
    transition: background-color 0.5s, transform 0.5s, font-weight  0.5s;
```